core/server/src/main/java/alluxio/proxy/FileSystemRestServiceHandler.java

```diff
@@ -49,13 +50,13 @@
 import javax.ws.rs.core.Response;
 
 /**
- * This class is a REST handler for file system client requests.
+ * This class is a REST handler for path resources.
  */
 @NotThreadSafe
-@Path(PathsRestServiceHandler.SERVICE_PREFIX)
+@Path(FileSystemRestServiceHandler.SERVICE_PREFIX)
 @Produces(MediaType.APPLICATION_JSON)
 @Consumes(MediaType.APPLICATION_JSON)
-public final class PathsRestServiceHandler {
+public final class FileSystemRestServiceHandler {
```

Comment:
I liked `PathsRestServiceHandler` better for consistency with StreamsRestServiceHandler, up to you